### DISCUSSION TITLE
240320

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#59] [팀회의]
## 멘토님 질문 사항

**Comment by caseV12** [comment]
## 논의사항

## 어제 한 일
x
## 오늘 할 일
어제 못한 일
많이 멀쩡해져서 빨리 진행하고 올리겠습니다. 이틀 못한만큼 열심히할게요 저 때문에 일정 늦어져서 죄송합니다 ㅠㅠ
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항

**Comment by MoonPillGyeom** [comment]
## 논의사항
- 지선님 pr댓글 봤는데 그럼 그냥 저 그대로 머지하면 되는건가요?
## 어제 한 일
- api테스트
## 오늘 할 일
- 모달이 많아져서 할일이 많아 졌어요 행복합니다 ㅎ.ㅎ

## 오늘의 일정 (코어 타임)
- 열심히 코딩하기
- 
## 멘토님 질문 사항
- x

**Comment by sobokki** [comment]
## 논의사항
## 어제 한 일
그룹필터 목데이터로 연동, 생성된 일정 캘린더에 넣기 시도중
## 오늘 할 일
생성된 일정 캘린더에 넣기 
필터기능에 맞춰서 선택한 일정만 보여주기 
## 오늘의 일정 (코어 타임)
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항
- 제 pr 함께 확인해요 ㅎ.ㅎ

## 어제 한 일
- svg 컴포넌트로 변경
   - 상수화한 컴포넌트 적용 문제 해결

## 오늘 할 일
- 데이터 작업
- 그룹 페이지 탭 기능
- 자유 게시판 ui
- 칸반보드 ui

## 오늘의 일정 (코어 타임)

## 멘토님 질문 사항
